tests/Aspire.Components.Common.Tests/Aspire.Components.Common.Tests.csproj

```diff
@@ -5,6 +5,10 @@
     <IsTestSupportProject>true</IsTestSupportProject>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\src\Components\Common\PasswordGenerator.cs" Link="PasswordGenerator.cs" />
+  </ItemGroup>
+  
```

Comment:
```suggestion
```

1. This is unnecessary.
2. This link is pointing to a place that doesn't exist. (Common vs Shared).